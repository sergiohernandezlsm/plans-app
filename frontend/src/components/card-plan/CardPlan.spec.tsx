import { shallow } from 'enzyme';
import CardPlan, { Props } from './CardPlan';

const props: Props = {
  card: {},
  selectionHandler: () => { },
}

describe('Card Component', () => {
  it('renders <Card /> component', () => {
    const wrapper = shallow(<CardPlan {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
