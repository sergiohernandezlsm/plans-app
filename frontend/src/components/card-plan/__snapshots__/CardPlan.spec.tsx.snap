// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component renders <Card /> component 1`] = `ShallowWrapper {}`;
